### PR TITLE
added level's and module's

### PR DESCRIPTION
**Pull request by Darina007**

### CODE REVIEW

**Review by Anlord241 — COMMENTED** [review]
Wouldn't it be better to use A1, A2... as level names?
I think it'll be much easier to work with them
Let's leave beatiful names to frontend

**Review by AntonMaksimenkau — CHANGES_REQUESTED** [review]
Please, add the appropriate enums for modules and levels.
Let's use a1, a2,... levels

**Review by AntonMaksimenkau — APPROVED** [review]

File: src/main/resources/db/changelog/changes/schema.xml [hunk 1]
```diff
@@ -376,6 +376,7 @@
     </changeSet>
 
     <changeSet id="add_level" author="Darina">
+        <validCheckSum>8:0b8ffa56b61fb87ce6cf127b376a30f2</validCheckSum>
```
Comment (AntonMaksimenkau) [inline src/main/resources/db/changelog/changes/schema.xml]: Not a valid changeset passing.
You need to revert a created changeset and create a new one

**Review by AntonMaksimenkau — CHANGES_REQUESTED** [review]

File: src/main/resources/db/changelog/changes/schema.xml (hunk 1 above)
Comment (Darina007) [reply src/main/resources/db/changelog/changes/schema.xml]: omg, ok😰

**Review by AntonMaksimenkau — APPROVED** [review]

**Review by maxim12321 — APPROVED** [review]

**Pull request merged**